### ISSUE TITLE
An idea for using yolov5

### ISSUE THREAD

**Issue by 15861329502** [ultralytics/yolov5#9901]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar bug report.

### Bug

Hello author, if I want to use yolov5 to identify people's attributes, such as not only identifying people, but also identifying people's gender, age, etc., how can I modify the code

### Are you willing to submit a PR?

- [x] Yes I'd like to help by submitting a PR!

**Comment by glenn-jocher** [comment]
If you have a dataset labelled with these attributes then you'd need to modify the dataloader, loss function, Detect() output count, visualization functions etc.

**Comment by 15861329502** [comment]
Thank you for your reply. I have finished this thought

**Comment by glenn-jocher** [comment]
@15861329502 thank you for your contribution! The YOLO community and the Ultralytics team greatly appreciate your efforts. If you have any further questions or need assistance, feel free to ask. Good luck with your project!
